app/services/scholars_archive/admin_set_select_service.rb

```diff
@@ -0,0 +1,9 @@
+module ScholarsArchive
+  class AdminSetSelectService
+    def self.select(model, select_options)
```

Comment:
I think this method would benefit from a brief comment about what it's doing.. I re-read it this morning and took a bit to jog my memory of how this relates to the select html element, and what this intends on returning.